stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/SingleLiveEvent.kt

```diff
@@ -0,0 +1,45 @@
+package io.getstream.chat.ui.sample.common
+
+import androidx.annotation.MainThread
+import androidx.annotation.Nullable
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Observer
+import io.getstream.chat.android.client.logger.ChatLogger
+import java.util.concurrent.atomic.AtomicBoolean
+
+class SingleLiveEvent<T> : MutableLiveData<T>() {
+
+    private val logger = ChatLogger.get("SingleLiveEvent")
+    private val mPending = AtomicBoolean(false)
```

Comment:
I'm not a big Hungarian notation fan and I don't think we are using it in our code, so I would recommend to remove `m` prefix